test/test_charge_drift_models.jl

```diff
@@ -81,6 +81,44 @@ end
     end
 end
 
+@timed_testset "Charge Trapping: ConstantLifetimeChargeTrappingModel" begin
```

Comment:
Can you also add tests for the inactive layer, especially how geometries are read in and parsed?